src/webui/api/synccontroller.cpp

```diff
@@ -72,6 +72,7 @@ namespace
     const QString KEY_PEER_FLAGS = u"flags"_s;
     const QString KEY_PEER_FLAGS_DESCRIPTION = u"flags_desc"_s;
     const QString KEY_PEER_IP = u"ip"_s;
+    const QString KEY_PEER_I2P_DEST = u"i2p_dest"_s;
```

Comment:
You'll need to bump web api version in src/webui/webapplication.h and document your change in WebAPI_Changelog.md.